plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs

```diff
@@ -401,8 +401,8 @@ genBadVerKey :: forall (a :: Type) .
 genBadVerKey = Gen.filter (isNothing . rawDeserialiseVerKeyDSIGN @a)
                           (Gen.bytes . Range.linear 0 $ 64)
```

Comment:
Since the ver key for ecdsa is now 33 bytes, I think you need to create an ecdsa version of `genBadVerKey` that generates length 33 keys.

Reply:
This shouldn't matter: the `Gen.filter` will reject anything that happens to deserialize irrespective of its length. This will occasionally generate things which are too long, but that's not actually a problem, since those are 'bad' by this definition.

Reply:
Oh right.. I mistakenly thought that this function is only supposed to generate inputs of the correct length.

I do wonder whether it is worth testing more cases where the input has the correct length, but that's a totally separate question.

Reply:
This is already _very_ heavily covered. If you examine the tests, you will see that case is tested extensively.